rfcs/template.md

```diff
@@ -0,0 +1,68 @@
+# Descriptive name for the proposal
+
+## Introduction
+
+Short description of the idea proposed with explained motivation. 
+
+The motivation could be:
+- Improved users experience for API changes and extensions. Code snippets to
+  showcase the benefits would be nice here.
+- Performance improvements with the data, if available.
+- Improved engineering practices.
+
+Introduction may also include any additional information that sheds light on
+the proposal, such as history of the matter, links to relevant issues and
+discussions, etc.
+
+## Proposal
+
+A full and detailed description of the proposal, with highlighted consequences.
+
+Depending on the kind of the proposal, the description should cover:
+
+- New use cases that are supported by the extension.
+- The expected performance benefit for a modification. 
+- The interface of extensions including class definitions or function 
+declarations.
+
+A proposal should include the alternatives that were considered with listed
+pros and cons. The alternatives should be clearly separated to make possible
+discussions clear.
+
+Pay close attention to the following aspects of the library:
+- API and ABI backwards compatibility. The library follows semantic versioning
+  so if any of those interfaces are to be broken, we need to state that
+  explicitly.
+- Performance implications, as performance is one of the main goals of the library.
+- Changes to the build system. While the library's primary building system is
+  CMake, there are some frameworks that may build the library directly from the sources.
+- Dependencies and support matrix: does the proposal brings any new
+  dependencies or affects the supported configurations.
```

Comment:
It is a question, right? Also, the `s` ending should be omitted in a question. Right?
```suggestion
- Dependencies and support matrix: does the proposal bring any new
  dependencies or affect the supported configurations?
```